index.js

```diff
@@ -94,12 +94,113 @@ module.exports = function (policyName) {
      */
     toString: function () {
       return policy.toString();
-    }
+    },
   };
 };
 
 module.exports.PasswordPolicy = PasswordPolicy;
 
+module.exports.createRulesFromSimpleConfig = createRulesFromSimpleConfig;
+
 module.exports.charsets = charsets;
 
-// module.exports.rulesToApply = rulesToApply;
+/**
+ * @typedef {Object} SimplePasswordConfig
+ * @property {number} [min_length] - Minimum password length (1-72)
+ * @property {Array<'uppercase'|'lowercase'|'number'|'special'>} [character_types] - Required character types
+ * @property {boolean} ['3of4_character_types'] - Whether to require 3 out of 4 character types (requires all 4 types to be specified)
+ * @property {'allow'|'disallow'} [identical_characters] - Whether to allow >2 identical consecutive characters
+ */
+
+/**
+ * Creates a PasswordPolicy rules configuration from a simplified configuration format.
+ * This provides an easier way to configure password policies without needing to
+ * understand the internal rule structure.
+ *
+ * @param {SimplePasswordConfig} config - Simplified password configuration object
+ * @returns {Object} Rules configuration object that can be passed to PasswordPolicy constructor
+ */
+function createRulesFromSimpleConfig(config = {}) {
+  var internalConfig = {};
+
+  // Apply defaults for any missing values
+  var defaults = {
+    min_length: 15,
+    character_types: [],
+    "3of4_character_types": false,
+    identical_characters: "disallow",
+  };
+
+  var normalizedConfig = {
+    min_length: config.min_length ?? defaults.min_length,
+    character_types: config.character_types ?? defaults.character_types,
+    "3of4_character_types": config["3of4_character_types"] ?? defaults["3of4_character_types"],
+    identical_characters: config.identical_characters ?? defaults.identical_characters,
+  };
+
+  // Validate min_length is within acceptable range
+  if (normalizedConfig.min_length < 1 || normalizedConfig.min_length > 72) {
+    throw new Error(
+      "min_length must be between 1 and 72"
+    );
+  }
+
+  // Handle min_length
+  if (normalizedConfig.min_length !== undefined) {
+    internalConfig.length = { minLength: normalizedConfig.min_length };
+  }
+
+  // Handle character_types and 3of4_character_types
+  var requiredTypes = normalizedConfig.character_types || [];
+  var require3of4 = normalizedConfig["3of4_character_types"];
+
+  // Validate 3of4_character_types prerequisite
+  if (require3of4) {
+    var hasAllFourTypes = ["lowercase", "uppercase", "number", "special"].every(function (type) {
```

Comment:
Thoughts on putting these value into an [enum](https://jsdoc.app/tags-enum)? Seems like there is a bit of reuse / redeclaration below and could be referenced in the JSDoc for this method